_sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md

```diff
@@ -1,14 +1,16 @@
 {:.do}
 ### Wear a facemask when in public
 
+Context: facemask here means any "face coverings", including scarves and bandanas.
+
 Studies have shown for Influenza that a mask may have a self-protection benefit. 
 The viron (complete virus particle) that causes COVID-19 is roughly the same size as the Influenza viron at 125 nanometers and it has the same mechanisms of entering the human body. 
 If you absolutely have to venture into public where other people may be, buy a cloth mask or make your own as US hospital chains are beginning to do. 
 
-Deaconess Health Systems shares a cloth mask design as part of their appeal for cloth masks (March 20): [PDF](https://www.deaconess.com/How-to-make-a-Face-Mask/Documents-Mask/Mask-Information) and [Video](https://youtu.be/9tBg0Os5FWQ).
+Many hospital groups have asked for cloth masks to be made by people with sewing machines. Deaconess Health System on the March 20 - ([PDF](https://www.deaconess.com/How-to-make-a-Face-Mask/Documents-Mask/Mask-Information) and [Video](https://youtu.be/9tBg0Os5FWQ)) and Providence health services as part of their "100 million mask" initiative on March 21 - ([Video](https://vimeo.com/399324367/13cd93f150)).
 
-Providence health services group shares a cloth mask design video (March 21) as part of their "100 million mask" initiative: [Video](https://vimeo.com/399324367/13cd93f150).
+By the third week of March, the countries of Vietnam, The Czech Republic (Czechia), and Slovakia are now operating a "wear a mask when in public" policy for their citizens and that includes homemade cloth masks. Austria is mandating that supermarket shoppers wear facemasks while shopping - see [Reuters article](https://www.reuters.com/article/us-health-coronavirus-austria-masks/austrian-supermarkets-hand-out-face-masks-before-they-become-compulsory-idUSKBN21J5Y9). South Africa [is encouraging facemask wearing too](https://www.dailymaverick.co.za/article/2020-04-01-the-great-mask-debate-policy-shifts-towards-masks-in-sa-and-elsewhere/) (since April 1).
 
-Vietnam, The Czech Republic, and Slovakia are now operating a "wear a mask when in public" policy for their citizens and that includes homemade cloth masks. Austrian [supermarkets are  handing out masks](https://www.reuters.com/article/us-health-coronavirus-austria-masks/austrian-supermarkets-hand-out-face-masks-before-they-become-compulsory-idUSKBN21J5Y9). South Africa [has begun encouraging mask wearing too](https://www.dailymaverick.co.za/article/2020-04-01-the-great-mask-debate-policy-shifts-towards-masks-in-sa-and-elsewhere/).
+Since April 2, the cities of New York and Los Angeles have mandated mask for people mixing in public.
```

Comment:
```suggestion
Since April 2, the cities of New York and [Los Angeles](https://twitter.com/ABC/status/1245670123823923200) have mandated mask for people mixing in public.
```